test_get_number_of_char (__main__.TestExercice) ... ok
test_get_number_of_words (__main__.TestExercice) ... ok
test_pair (__main__.TestExercice) ... ok
test_remove_third_char (__main__.TestExercice) ... ok
test_replace_char (__main__.TestExercice) ... FAIL

======================================================================
FAIL: test_replace_char (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 49, in test_replace_char
    'Erreur dans le remplacement de caractère'
AssertionError: Lists differ: ['hey jaz!', 'aaaaabz', '01234z'] != ['hey yad!', 'bbbbbb', 'a1234']

First differing element 0:
'hey jaz!'
'hey yad!'

- ['hey jaz!', 'aaaaabz', '01234z']
+ ['hey yad!', 'bbbbbb', 'a1234'] : Erreur dans le remplacement de caractère

----------------------------------------------------------------------
Ran 5 tests in 0.001s

FAILED (failures=1)
